packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -769,7 +769,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             pendingRuntimeState.savedOps = [];
         }
 
-        await runtime.getSnapshotBlobs();
+        // Initialize the base state of the runtime before it's returned.
+        await runtime.initializeBaseState();
```

Comment:
@wes-carlson Could you please take a look at this part of the PR since you added this code that I modified.

Reply:
`initializeBaseStateFromSnapshot`?